PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,19 @@
-Thank you for submitting a pull request to the WrongSecrets app!
+Thank you for submitting a pull request to the WrongSecrets app! Please link any relevant issues.
```

Comment:
maybe explain how to do that with an example? (e.g. #333 where 333 is your issue number)